src/Penrose/Style.hs

```diff
@@ -230,6 +235,7 @@ data Expr
     | AFloat AnnoFloat
     | StringLit String
     | BoolLit Bool
+    | EVar LocalVar
```

Comment:
@wodeni Could we make local vars part of `Path` instead of `Expr`? I know that we missed this in the Style 1.1 grammar doc, but the rationale is that some of the expressions that currently only involve `Path` should be writeable using local vars. 

Example: `layer v.shape above perpMark` (where `perpMark` is a local var that is assigned to a shape)